internal/evaluator/rego.go

```diff
@@ -118,6 +159,30 @@ func ociBlob(bctx rego.BuiltinContext, a *ast.Term) (*ast.Term, error) {
 	return ast.StringTerm(blob.String()), nil
 }
 
+func purlIsValid(bctx rego.BuiltinContext, a *ast.Term) bool {
+	uri, ok := a.Value.(ast.String)
+	if !ok {
+		return false
+	}
+	instance, err := packageurl.FromString(uri)
+	if err != nil {
+		return false
+	}
+	return true
```

Comment:
I wonder if you need something like this: 

```go
return ast.BooleanTerm(true)
```

Reply:
Done!